WebPhapp/backend/server.js

```diff
@@ -139,6 +139,59 @@ app.get('/api/v1/prescriptions/:patientID', (req,res) => {
                 ' prescription(s) for patient ID ' + patientID.toString());
 });
 
+/*
+Edits a prescription for a given prescription ID. The prescriptionID is used in order to get the rest of the data for the prescription. The rest of the data points are alterable values.
+
+ Expects on object of shape:
+{
+  prescriptionID,
+  quantity,
+  daysFor,
+  refillsLeft,
+  dispenserID
+}
+
+  Directly in terminal:
+    >>> curl 'http://localhost:5000/api/v1/prescriptions/edit' -H 'Acceptapplication/json, text/plain, /*' -H 'Content-Type: application/json;charset=utf-8' --data '{"prescriptionID": 3,"drugID":0,"quantity":1,"daysValid":0,"refills":0,"dispenserID":0}'
+
+    To be used in an axios call:
+        .post("/api/v1/prescription/edit",{
+            prescriptionID: 0,
+            ....
+        }
+*/
+
+app.post('/api/v1/prescriptions/edit',(req,res) => {
```

Comment:
This is great, Max. I'll be able to go into this function and easily extend it to work with BC functions.